### PR TITLE
feat(database): use sqlite instead of json to store messages

### LINKED ISSUE #35: Feature-Request: Use database for storing the conversion

### PR DESCRIPTION
**Pull request by AceFire6**
Store all messages in a SQLite database instead of a file-based JSON file structure.

Closes #35

### CODE REVIEW

**Comment by Haider8** [comment]
@AceFire6 Use ORM like sql alchemy instead of writing the database queries manually.

**Comment by iamareebjamal** [comment]
peewee is much simpler and lightweight - http://docs.peewee-orm.com/en/latest/index.html

**Comment by AceFire6** [comment]
Sure thing @Haider8. I left it out initially because I wasn't sure if you were okay adding additional dependencies for this. I'll use peewee 👍

File: tmessage/cli.py [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-from datetime import datetime
-import paho.mqtt.client as mqtt
 from colorama import init, deinit, Fore, Back, Style
+import paho.mqtt.client as mqtt
 import tmessage.auth as auth  # auth.py
 
 # Initialize colorama
+from tmessage.db import store_messages
+
 init()
```
Comment (Haider8) [inline tmessage/cli.py]: `# Initialize colorama` line will come above `init()`

**Pull request merged**